packages/themes/patches/bootstrap-sass+3.4.3+001+encode-button-loading-text.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/node_modules/bootstrap-sass/assets/javascripts/bootstrap.js b/node_modules/bootstrap-sass/assets/javascripts/bootstrap.js
+index 170bd60..fa6e67f 100644
+--- a/node_modules/bootstrap-sass/assets/javascripts/bootstrap.js
++++ b/node_modules/bootstrap-sass/assets/javascripts/bootstrap.js
+@@ -211,7 +211,7 @@ if (typeof jQuery === 'undefined') {
+ 
+     // push to event loop to allow forms to submit
+     setTimeout($.proxy(function () {
+-      $el[val](data[state] == null ? this.options[state] : data[state])
++      $el[val](data[state] == null ? LABKEY.Utils.encodeHtml(this.options[state]) : LABKEY.Utils.encodeHtml(data[state]))
```

Comment:
This does mean our themes rely on our `clientapi` which is not strictly upheld by the client-side dependency declarations at the moment. That said, effectively all pages receive the `clientapi` lib package.